files/templates/alertmanager/alertmanager.yaml

```diff
@@ -30,6 +30,11 @@ route:
     matchers:
     - alertname="PrometheusMetaOperatorReconcileErrors"
     continue: false
+  - receiver: blackhole
+    matchers:
+    - alertname="ClusterUnhealthyPhase"
+    - name=~"t-.*"
```

Comment:
Are you sure about `name` ?
Isn't it rather `cluster_id` ?

Reply:
`cluster_id` for this alert is the MC. `name` is the WC ID for this alert